orbs/shared/jobs/test.yaml

```diff
@@ -53,6 +57,7 @@ executor:
 environment:
   AWS_REGION: << parameters.aws_region >>
   PRE_SETUP_SCRIPT: << parameters.pre_setup_script >>
+  GO_TEST_TIMEOUT: << parameters.go_test_timeout >>
```

Comment:
so i take it this is used in a script somewhere when we run the tests?

Reply:
Yup.
https://github.com/getoutreach/devbase/blob/be1b62d86f8295ea542f9adecdf7ace7990cf5bc/shell/test.sh#L103-L106